README.md

```diff
@@ -2,7 +2,7 @@
 
 [![build status](https://github.com/ReproNim/neurodocker/actions/workflows/pull-request.yml/badge.svg)](https://github.com/ReproNim/neurodocker/actions/workflows/pull-request.yml)
 [![docker pulls](https://img.shields.io/docker/pulls/repronim/neurodocker.svg)](https://hub.docker.com/r/repronim/neurodocker/)
-[![docker pulls](https://img.shields.io/docker/pulls/kaczmarj/neurodocker.svg)](https://hub.docker.com/r/kaczmarj/neurodocker/)
+[![docker pulls](https://img.shields.io/docker/pulls/repronim/neurodocker.svg)](https://hub.docker.com/r/repronim/neurodocker/)
```

Comment:
can you remove this badge? previously we had two badges for docker pulls, one from the repronim namespace and one from my personal kaczmarj namespace. i included my personal namespace because for a long time, most people pulled images from there. in other words, we should only have a docker badge for the repronim namespace.

Reply:
done